Scraper/utils.py

```
@@ -0,0 +1,72 @@
+def load_existing_books():
+    """
+    Load existing books from JSON file
+    """
+
+    if not os.path.exists(JSON_FILE):
+        return []
+
+    with open(JSON_FILE, "r") as file:
+        try:
+            return json.load(file)
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)